package.json

```diff
@@ -12,6 +12,7 @@
   },
   "dependencies": {
     "@next/font": "13.1.6",
+    "@sentry/nextjs": "^7.37.2",
```

Comment:
We won't use this yet since it'll take some more setup, perhaps down the line we will use it though. So for now `yarn remove @sentry/nextjs`